.claude/skills/docx/ooxml/scripts/validation/docx.py

```
@@ -0,0 +1,274 @@
+                                # Show a preview of the text
+                                text_preview = (
+                                    repr(text)[:50] + "..."
+                                    if len(repr(text)) > 50
+                                    else repr(text)
+                                )
+                                errors.append(
+                                    f"  {xml_file.relative_to(self.unpacked_dir)}: "
+                                    f"Line {elem.sourceline}: w:t element with whitespace missing xml:space='preserve': {text_preview}"
+                                )
+
+            except (lxml.etree.XMLSyntaxError, Exception) as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)